### ISSUE TITLE
Enhancement Request - Ability to Print A List of the Teams Competing

### ISSUE THREAD

**Issue by tfullmer76** [FIRST-Tech-Challenge/scorekeeper#346]
Hello,

I would really love the ability to print a list containing all of the teams in the system (with their team number, name, city, state & affiliation). I get asked for one  all day long during a tournament (30+ teams).  Highlighting the teams listed, in ADD TEAMS on the event Dashboard, and selecting print does not work. To much of the information is missing.  I know the announcer report has the team info, but that is only after matches are created and has the matches with it.

Thanks!

**Comment by tfullmer76** [comment]
Hello Craig.

We haven't used either yet, but thank you for the information!

**Comment by adwiii** [comment]
A list of teams competing has been added in 0.7.5. You can find it under the Event Reports link under the Event Info heading.

**Comment by jvens** [comment]
The original request was for a list of teams including their City, State, & Affiliation.  It would be nice to add these columns to the team list report.  

It would also be a nice to have feature to turn some of these columns on or off (similar to the match start time in the match list) to customize the report depending on what it is being printed for.

**Comment by tfullmer76** [comment]
Thank you!

**Comment by adwiii** [comment]
@jvens can you open a new issue if that is still a requested feature?
